browse/controllers/list_page/paging.py

```diff
@@ -0,0 +1,65 @@
+"""Paginiation links for listing pages."""
+
+
+from typing import Any, Dict, List, Union
+import math
+from flask import url_for
+
+
+def paging(count: int, skipn: int, shown: int, context: str, subcontext: str) \
+    -> List[Dict[str, Union[str, int]]]:
+    """Get paging links."""
+    B = 3  # num of buffer pages on each side of current
```

Comment:
Unless these are widely recognized/conventional symbols, would prefer to avoid single-character names. A quick glance suggests that it would not be too unwieldy to call this ``number_of_buffer_pages``, for example, which would increase legibility.